scripts/validation.ipynb

```diff
@@ -93,7 +93,7 @@
     "    packages = 'pyyaml pandas numpy matplotlib'\n",
     "    ret = os.system(f'conda install {packages}')\n",
     "    if ret != 0:\n",
-    "        os.system(f'pip install {packages}')"
+    "        os.system(f'{sys.executable} -m pip install {packages}')"
```

Comment:
you shouldn't call pip or python directly in case the user path picks a different python than the one running the notebook.